### DISCUSSION TITLE
Email parser

### DISCUSSION THREAD

**Discussion by rosquillas** [firefly-iii/firefly-iii#8551] [General]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

Would it be possible to write an email parser that would take te payments notifications via email and import them? or move them into a table for import? I'm thinking even of using an LLM to 'understand' the email.

### Solution

It would make possible to keep Firefly III up-to-date on every transaction that has an email alert attached to it.

### What are alternatives?

No alternatives to this tmk

### Additional context

My bank issues an email on every credit card transaction I make. This information could be used to feed Firefly III

**Comment by JC5** [comment]
That's certainly possible! But I'm not going to make it part of Firefly III 😅 .

Do you need pointers? There's an API you can use to send a transaction to Firefly III, but the parsing from email is something you'll have to do yourself.

**Comment by rosquillas** [comment]
Thanks for the reply, I'll give it a try myself.
